openeo/internal/processes/parse.py

```diff
@@ -127,6 +127,12 @@ def from_json_file(cls, path: Union[str, Path]) -> Process:
         with Path(path).open("r") as f:
             return cls.from_json(f.read())
 
+    def get_parameter(self, parameter_id: str) -> Union[dict, list]:
+        for parameter in self.parameters:
+            if parameter.name == parameter_id:
+                return parameter.schema.schema
```

Comment:
I would return the whole parameter instead of just its schema to be more generic
```suggestion
                return parameter
```